doc/ref_model/chapters/chapter02.md

```diff
@@ -266,16 +266,44 @@ By trying to sort workloads into different categories based on the requirements
 
-Based on the above analysis, following cloud infrastructure profiles are proposed (also shown in **Figure 2-1** below)
+Infrastructure profiles are used to tag infrastructure (such as hypervisor hosts, or Kubernetes worker nodes) and associate it with a set of capabilities exploitable by the workloads. *Workload profiles* are requirements expressed as workload metadata, indicating what kind of infrastructure they must run on to achieve functionality and/or the intended level of performance.
+Two profile *layers* are proposed:
+
+- The top level **profiles** represent macro-characteristics that partition infrastructure into separate pools, i.e.: an infrastructure object can belong to one and only one profile, and workloads can only be created using a single profile. Workloads requesting a given profile **must** be instantiated on infrastructure of that same profile.
+- For a given profile, **flavours** represent small deviations from (or further qualification, such as sizing) the profile that do not require partitioning the infrastructure into separate pools, but that have specifications with a finer granularity of the profile. Flavours can be *optionally* requested by workloads that want a more granular control over what infrastructure they can run on, i.e.: an infrastructure resource can have **more than one flavour** attached to it, and workloads can request VMs to be instantiated on infrastructure with a certain flavour. Workloads requesting a given flavour **must** be instantiated on infrastructure with that same flavour. It is allowed to instantiate workloads on infrastructure with more flavours than what is requested, as long as the minimum requirements are satisfied.
+
+### 2.4.1 Infrastructure profiles (top-level)
+
+Based on the above analysis, the following cloud infrastructure profiles are proposed (also shown in **Figure 2-1** below)
 - **Basic**: for Workloads that can tolerate resource over-subscription and variable latency.
-- **Network Intensive**: for Workloads that require predictable computing performance, high network throughput and low network latency.
+- **High Performance**: for Workloads that require predictable computing performance, high network throughput and low network latency.
 
 <p align="center"><img src="../figures/ch02_infra_profiles.PNG" alt="infra_profiles" title="Infrastructure Profiles" width="100%"/></p>
 <p align="center"><b>Figure 2-1:</b> Infrastructure profiles proposed based on VNFs categorisation.</p>
 
-In **[Chapter 4](./chapter04.md)** these **B (Basic)** and **N (Network intensive)** infrastructure profiles will be defined in greater detail for use by workloads:
+In **[Chapter 4](./chapter04.md)** these **B (Basic) Performance** and **H (High) Performance** infrastructure profiles will be defined in greater detail for use by workloads.
+
+Profiles partition the infrastructure: an infrastructure object (host/node) **must** have one and only one profile associated to it.
```

Comment:
do we need to state that while an infrastructure object (host/node) has only one profile associated with it, the infrastructure object by the nature of hierarchical profiles implicitly is associated with "ancestor" profiles.

Reply:
is this ok as is?